### DISCUSSION TITLE
Spawn/drop item at cursor?

### DISCUSSION THREAD

**Discussion by SHACKTINGG** [Pryaxis/TShock#2070] [Q&A] [answered]
Hey everyone, I'm wondering if there's a command / plugin that I'm missing that allows you to spawn an item at your cursor? I.e if I use the command, click and my cursor is hovering mid-air above a platform, the item will spawn at my cursor and fall to the ground.

Thanks guys, hope everyone's doing well and staying safe.

**Comment by bartico6** [comment, marked as answer]
The server doesn't know where your cursor is most of the time, only that you're swinging your weapon.
There is no such plugin either, so you'd probably have to make one.

If you want to determine where the user clicked you might need to either use grand design (which will allow you to set the spawn position with one-block precision *anywhere on your screen*) or any pickaxe (but that'll require a block to hit with it, and close proximity)

**Reply by SHACKTINGG** [reply]
Do you know if there's any way to use the spawnmob command to spawn the mobs in a less random area? Like is there anyway to further 'control' where they spawn?

**Reply by bartico6** [reply]
The stock spawnmob command? No, you can't control where things spawn. You need a custom plugin for that too.
